.github/workflows/publish_draft_releases.yml

```diff
@@ -188,7 +188,7 @@ jobs:
       - if: ${{ needs.check-for-runtime-upgrade.outputs.do-versions-match == 'false' }}
         name: fetch and chmod polkadot
         run: |
-          curl -L -o $HOME/.local/bin/polkadot https://github.com/paritytech/polkadot/releases/download/v0.9.18/polkadot
+          curl -L -o $HOME/.local/bin/polkadot https://github.com/paritytech/polkadot/releases/download/v0.9.22/polkadot
```

Comment:
Can we make this a yaml config? (not necessarily have to be addressed in this PR)

Reply:
I created an issue to track this: https://github.com/Manta-Network/Manta/issues/670